docs/whatsnew/7.0.rst

```diff
@@ -273,6 +274,15 @@ value in ``Unit.to`` and have those arrays not be converted to Numpy arrays:
 
 Note that it is not yet possible to use ``Quantity`` with dask arrays directly.
 
+.. _whatsnew_7_0_modeling_speedup:
+
+Performance improvements in astropy.modeling
+============================================
+
+There have been significant improvements to the performance of fitting in
```

Comment:
Do we want to say to the nonlinear fitters?

Reply:
Done